openfisca_core/scripts/tools/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
```

Comment:
I'd rename the directory containing this file `assets` rather than `tools`

Reply:
With Chrome, I can't simply open the `index.html` page on my browser, as the request to `performance.json` is blocked by the CORS policy:

I'm able to get the graph only with a small HTTP server like `python -m http.server 8000`

How to open the graph should be documented, at least in the Changelog and PR description. 

![image](https://user-images.githubusercontent.com/11834997/61832213-69569c80-ae3e-11e9-9c09-10f20df86e43.png)

Reply:
👍 It will be copied to the CHANGELOG. 
> It's already in this PR description.